### DISCUSSION TITLE
Can't Sign in to Play Store

### DISCUSSION THREAD

**Discussion by rzrabbi** [LSPosed/MagiskOnWSALocal#444] [Q&A] [answered]
I'm trying to sign in to Play Store, but I keep getting an error message that says, "There is a problem communicating with Google server. Try again later." I've tried several times, but the same error message keeps popping up. 
I followed all the steps for installation. I selected "MindTheGapps" during the installation. What should I do now?

![Screenshot 2023-03-25 142935](https://user-images.githubusercontent.com/76894655/227706450-cfa2463d-b39b-4b5e-9118-7ddecd5f4fbf.png)

**Comment by Zeo7coder** [comment, marked as answer]
If you're having trouble signing in to the Google Play Store, a simple solution that often works is to restart your PC. This can help clear out any temporary issues that may be preventing you from accessing the Play Store. It's a common issue that many people face from time to time, including myself. However, I can confirm that restarting my device did solve the problem for me.
